.travis.yml

```diff
@@ -24,9 +28,13 @@ matrix:
     compiler: gcc
 
 before_install:
-  - if [ $TRAVIS_OS_NAME == linux ]; then sudo apt-get install -qq cmake3 cppcheck clang-tidy-3.8 libfreetype6-dev libpng-dev; fi
+  - if [ $TRAVIS_OS_NAME == linux ]; then sudo apt-add-repository ppa:cginternals/ppa -y; fi
+  - if [ $TRAVIS_OS_NAME == linux ]; then sudo apt-add-repository ppa:cginternals/backports-ppa -y; fi
+  - if [ $TRAVIS_OS_NAME == linux ]; then sudo apt-get update -qq; fi
+  - if [ $TRAVIS_OS_NAME == linux ]; then sudo apt-get install -qq cmake3 cppcheck clang-tidy-3.8 libfreetype6-dev libpng-dev qt5-default libglm-dev libglbinding-dev libglobjects-dev; fi
   # libpng is already installed by default
-  - if [ $TRAVIS_OS_NAME == osx ]; then brew update && brew install cppcheck freetype; fi
+  - if [ $TRAVIS_OS_NAME == osx ]; then brew tap cginternals/homebrew-cginternals; fi
+  - if [ $TRAVIS_OS_NAME == osx ]; then brew update && brew install cppcheck freetype qt5 glm glbinding globjects; export CMAKE_PREFIX_PATH="$CMAKE_PREFIX_PATH;$(brew --prefix qt);$(brew --prefix glbinding);$(brew --prefix globjects)"; fi
```

Comment:
I think this could be turned into a multi line string like so (to avoid repeating the if statement making the whole thing more readable):
```yaml
- |-
  if [ ... ]; then
    ...;
    ...;
  fi
```

The `|` starts a multi line string, and the additional `-` cuts out trailing newlines from the string (not necessarily needed).

Reply:
Cool, thanks!